Info.plist

```diff
@@ -24,18 +26,42 @@
 	<dict>
 		<key>tsInputModeListKey</key>
 		<dict>
-			<key>im.rime.inputmethod.Squirrel</key>
```

Comment:
This InputModeID was incorrect. It should be bundle name plus an identifier for the input mode (used to be `.Rime`).